src/core/core.layoutService.js

```diff
@@ -81,6 +81,15 @@ module.exports = function(Chart) {
 		},
 
 		/**
+		 * Sets (or updates) options on the given `item`.
+		 */
```

Comment:
In the interest of completeness, should we add JSDoc comments for the parameters here?